src/tests/miscellaneous/mock-adb/app/bin.js

```diff
@@ -92,6 +96,10 @@ async function main() {
 
     result.input = process.argv;
     result.inputCommand = inputCommand;
+
+    fs.writeFileSync(path.join(testLogsDir, 'adb.log'), `ADB ${inputCommand}\n`, {
```

Comment:
Would be it practical to make the test logging behavior controlled by a flag in the mock-adb-config.json file? It's probably not a huge deal to log in 2 places, but it feels cleaner to only log it when we need it

Reply:
Per offline conversation, leaving this as is because we're already doing a lot of logging that can't be disabled and the new logs live alongside the old logs.